Altis_Life.Altis/core/functions/TON/fn_clientGangLeader.sqf

```diff
@@ -0,0 +1,9 @@
+/*
+    File: fn_clientGangLeader.sqf
+    Author: Bryan "Tonic" Boardwine
+
+    Description: appoints player as gang leader
+*/
+player setRank "COLONEL";
+(group player) selectLeader player;
```

Comment:
Will locality ever serve at issue here? (Prior this was on all units).

Reply:
afaik, yes -> https://community.bistudio.com/wiki/selectLeader

has to be executed where unit is local

Reply:
But is the group?